docs: Improve readability of page elements by LLM and AI tools (#47275)
supabase/supabase: apps/docs/content/guides/platform/read-replicas.mdx

    When your database starts slowing down, you face a choice: make your existing database bigger (scale vertically), or spread the load across multiple databases (scale horizontally). Both approaches work. Neither is universally correct. The right answer depends on your workload, your budget, and where the bottleneck actually is.

    ```mermaid
    flowchart TD
      A[Database slowing down] --> B{CPU above 70% sustained?}
      B -->|No| C[Monitor, do not scale yet]
      B -->|Yes| D{Queries optimized? Indexes in place?}
      D -->|No| E[Run EXPLAIN ANALYZE<br/>Add missing indexes<br/>Optimize first]
      E --> D
      D -->|Yes| F{Workload 80%+ reads?}
      F -->|No| G[Upgrade compute<br/>Replicas will not help writes]
      F -->|Yes| H{Already at 16XL?}
      H -->|Yes| I[Read Replicas<br/>Only horizontal option left]
      H -->|No| J{Need workload isolation<br/>or geo-distribution?}
      J -->|Yes| K[Read Replicas]
      J -->|No| L[Either works<br/>Compute is simpler<br/>Replicas scale further]
    ```

>     We recommend not scaling until CPU is sustained above 70%. Once it is, confirm your queries are already optimized and properly indexed (use `EXPLAIN ANALYZE`) before adding hardware. If your workload is less than roughly 80% reads, upgrade compute. Read Replicas only serve reads and won't help writes. If it's read-heavy, Read Replicas become the right choice once you've reached the largest compute size (16XL), or earlier if you need workload isolation or geographic distribution. Below 16XL with no isolation need, either option works: compute is simpler, Read Replicas scale further.

Reviewer: ```suggestion
    Supabase recommends not scaling until CPU is sustained above 70%. After it is, confirm your queries are already optimized and properly indexed with `EXPLAIN ANALYZE` before adding hardware. If your workload is less than roughly 80% reads, upgrade compute. Read Replicas only serve reads and won't help writes. If it's read-heavy, Read Replicas become the right choice after you reach the largest compute size of 16XL or earlier if you need workload isolation or geographic distribution. Below 16XL with no isolation need, either option works. While compute is simpler, Read Replicas scale further.
```

Is there a convention on "We recommend"? I've heard debate about this, as it is unclear who 'we' are. The convention settled on was "[company] recommends" as it has more authority and isn't vague.

`Once -> After` is for localization. `Once` has a double meaning.

Overall, find this paragraph wordy and confusing.
